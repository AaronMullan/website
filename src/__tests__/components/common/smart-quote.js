import React from 'react'
import renderer from 'react-test-renderer'
import SmartQuote from '~components/common/smart-quote'

describe('Components : Common: Smart quote', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<SmartQuote>Don&#8217;t worry</SmartQuote>)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
