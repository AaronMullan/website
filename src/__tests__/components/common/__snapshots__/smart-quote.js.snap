// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components : Common: Smart quote renders correctly 1`] = `"Donâ€™t worry"`;
